object_tracker_0/src/inference.py

```diff
@@ -69,7 +70,7 @@ def calculate_croppings(pil_frame, sub_frame_height, sub_frame_width):
             croppings.append((left, top))
     return croppings
 
-def do_object_detection(model, processor, pil_frame, tiling = False):
+def do_object_detection(model, processor, pil_frame, tiling = True):
```

Comment:
I forgot to revert it, but we can leave it set to True by default until we fix the config